sphinx_click/ext.py

```diff
@@ -218,7 +218,16 @@ def _format_envvar(
 
 def _format_envvars(ctx: click.Context) -> ty.Generator[str, None, None]:
     """Format all envvars for a `click.Command`."""
-    params = [x for x in ctx.command.params if x.envvar]
+
+    auto_envvar_prefix = ctx.auto_envvar_prefix
```

Comment:
This is [present in 7.x too](https://click.palletsprojects.com/en/7.x/options/#dynamic-defaults-for-prompts) so we don't need to worry about backwards compatibility :+1: